win32ss/gdi/gdi32/objects/dc.c

```diff
@@ -1057,6 +1057,10 @@ SetBkMode(
     _In_ HDC hdc,
     _In_ int iBkMode)
 {
+    /* Avoid bad mode setting */
+    if (iBkMode != TRANSPARENT)
+        iBkMode = OPAQUE;
+	
     PDC_ATTR pdcattr;
     INT iOldMode;
 
```

Comment:
Suggestion for keeping the code C-compliant
```suggestion
    PDC_ATTR pdcattr;
    INT iOldMode;

    /* Avoid bad mode setting */
    if (iBkMode != TRANSPARENT)
        iBkMode = OPAQUE;

```